frontend/src/features/properties/hooks/useParcelLayerData.tsx

```diff
@@ -134,7 +134,11 @@ const getAdminAreaFromLayerData = (
     const splitLayerMunicipality = layerMunicipality.split(',');
     if (splitLayerMunicipality.length === 2) {
       const formattedLayerMunicipality = `${splitLayerMunicipality[1].trim()} ${splitLayerMunicipality[0].trim()}`;
-      return _.find(administrativeAreas, { name: formattedLayerMunicipality });
+      let match = _.find(administrativeAreas, { name: formattedLayerMunicipality });
+      if (!match) {
+        match = _.find(administrativeAreas, { name: splitLayerMunicipality[0].trim() });
```

Comment:
What is the `splitLayerMunicipality`?

Reply:
I'm attempting to match the location from parcel map with our locations as best I can, so I'm splitting the municipality from the layer and using the split string to try and match one of our locations.